tests/unit/js/examples.js

```diff
@@ -0,0 +1,70 @@
+const fieldset = `<fieldset id="identifiers">
```

Comment:
Rename file to `html-test-data.js` . This file contains HTML used for tests, so let's make the name specific. Examples is very vague.

Reply:
Done